src/typescript/azure-openapi-validator/rules/Rpaas_CreateOperationAsyncResponseValidation.ts

```diff
@@ -0,0 +1,60 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { MergeStates, OpenApiTypes, rules } from "../rule"
+export const Rpaas_CreateOperationAsyncResponseValidation: string = "Rpaas_CreateOperationAsyncResponseValidation"
+
+rules.push({
+  id: "R4011",
+  name: Rpaas_CreateOperationAsyncResponseValidation,
+  severity: "error",
+  category: "ARMViolation",
+  mergeState: MergeStates.individual,
+  openapiType: OpenApiTypes.arm,
```

Comment:
the openapType should be OpenApiTypes.rpass

Reply:
I suppose that is SubType

Reply:
export interface Rule {} needs to have openapiSubType property. Could you please add it ?
The idea is that openapitype is still ARM, since these are all ARM apis rules.
But, sub type RPaaS indicates that these are ARM service (RPs) that are onboarded to RPaaS platform.

Reply:
changed this to rpaas per offline discussion.